vernac/comFixpoint.mli

```diff
@@ -80,12 +78,13 @@ val interp_fixpoint :
 (** Registering fixpoints and cofixpoints in the environment *)
 (** [Not used so far] *)
 val declare_fixpoint :
-  locality -> polymorphic ->
+  check_guard -> locality -> polymorphic ->
```

Comment:
The type signature doesn't correspond to the implementation (compilation is broken).
Can you make these three first parameters named ones since you are changing the API anyways?
BTW, what is this "Not used so far" comment above?